src/java/org/apache/cassandra/cql3/constraints/ColumnConstraints.java

```diff
@@ -49,10 +49,17 @@ public class ColumnConstraints extends ColumnConstraint<ColumnConstraints>
 
     public ColumnConstraints(List<ColumnConstraint<?>> constraints)
     {
-        super(null);
         this.constraints = constraints;
     }
 
+    @Override
+    public void setColumnName(ColumnIdentifier columnName)
```

Comment:
we make sure it is set everywhere for all constraints the same way and for this "encapsulating" constraint as well